examples/cppwin/TensorflowTTSCppInference/Voice.cpp

```diff
@@ -0,0 +1,99 @@
+#include "Voice.h"
+#include "ext/ZCharScanner.h"
+const std::vector<std::string> Phonemes = { "AA","AA0","AA1","AA2","AE","AE0","AE1","AE2","AH","AH0","AH1","AH2","AO","AO0","AO1",
```

Comment:
I also considered that before deciding against it; this is just an example and if someone wants to adapt it to their dataset then they should take it then change it. It's not designed to be a one size fits all solution, but something that people can use as a base for their project.

Reply:
yeah :)) it's just example :D, we do not need make it general :D.

Reply:
Maybe but every other example right now can be used in very easy way just changing few things in scripting part 
I think we should move in this direction. (In future prepro there will be mapper saving so you can load phonemes and other mappers from json files just like configs right now :P)

Still im not following any C++ standards so this is a question for  @candlewill

Reply:
:))) for this case, i just to make sure it can be run then i will approve merge =)))))))))))))))).

Reply:
>In future prepro there will be saving all needed mappers so you can load phonemes and other mappers from json files just like configs right now

Why not leave it like this right now then later when those preprocessing steps are implemented I make it so that the program can load exported dictionaries? Because another issue is to agree on a format for storing the characters and IDs.

Reply:
We can just wait for @candlewill to check it and whole pr :P